docs/src/getting_started/toolsinstallation.md

```diff
@@ -2,12 +2,14 @@
 
 ## [Julia REPL](@id julia-repl)
 
-A REPL, or read-eval-print loop, represents a language "shell" or program that allows the user to communicate with the
-programming language interactively with immediate feedback. This allows for exploring the language more deeply than via
+A REPL, or read-eval-print loop, represents a language "shell" or program that
+allows the user to communicate with the programming language interactively with
+immediate feedback. This allows for exploring the language more deeply than via
 a script, with rapid prototyping being much quicker and easier to perform.
 
-The Julia REPL is included by default in every Julia installation and boasts a number of features to the point that it
-alone can almost be considered a proper IDE. It should appear the moment you enter `julia` into your terminal:
+The Julia REPL is included by default in every Julia installation and boasts a
+number of features to the point that it alone can almost be considered a proper
+IDE. It should appear the moment you enter `julia` into your terminal:
```

Comment:
Are we recommending people install julia via `juliaup`? I'm not sure how accessible the idea of a terminal is to windows users in particular, and I don't know how `juliaup` handles this on windows if installed through the app store. May be worth confirming.